src/pathfinding_service/api.py

```diff
@@ -471,7 +471,10 @@ def get(self, checksummed_address: str) -> Tuple[Dict[str, Union[str, PeerCapabi
                 capabilities = user_manager.get_address_capabilities(address)
                 return {"user_id": user_id, "capabilities": capabilities}, 200
 
-        raise exceptions.AddressNotOnline(address=checksummed_address)
+        offline_since = datetime.now() - user_manager.seen_offline_at(address)
+        raise exceptions.AddressNotOnline(
+            address=checksummed_address, seen_offline_since=offline_since.seconds
```

Comment:
I believe using `datetime.timedelta.seconds` is misleadingly wrong:
```python
>>> (datetime(year=2012, month=5, day=1) - datetime(year=2010, month=5, day=1)).seconds
0
```

Reply:
Seems like the correct method to use here would be `total_seconds()`

```python
>>> (datetime(year=2012, month=5, day=1) - datetime(year=2010, month=5, day=1)).total_seconds()
63158400.0
```

Reply:
:+1: